oslo/transformers/mapping_utils.py

```diff
@@ -83,7 +83,7 @@ class _TensorParallelMappingForHuggingFace(_ParallelMappingForHuggingFace):
         ],
         "Bert": [
             Column("query", "key", "value", "intermediate.dense"),
-            Column("pooler.dense", gather_output=True),
+            Column("pooler.dense", "transform.dense", gather_output=True),
```

Comment:
Why should we add this?

Reply:
At the code below, it is also dense layer before head so I think it should be splitted.
https://github.com/huggingface/transformers/blob/ccc089780415445768bcfd3ac4418cec20353484/src/transformers/models/bert/modeling_bert.py#L664

It may be slower but requires lower memory same as tensor parallel intends.